packages/SidePanel/src/components/Dialog/Dialog.styles.js

```diff
@@ -0,0 +1,79 @@
+import { css, keyframes } from "styled-components";
+import tokens from "@paprika/tokens/lib/tokens";
+
+const space = Number.parseInt(tokens.space, 10);
+
+const gapChildPanel = space * 7;
+const childPanel = css`
+  border-bottom-left-radius: ${space / 2}px;
+  border-top-left-radius: ${space / 2}px;
+  height: calc(100% - ${gapChildPanel + 80}px);
```

Comment:
What is `80`?

Reply:
To adjust the position of the child sidepanel so is not exactly at the same level of the Parent, I totally picked by `eye` there is not a hard guideline for this, later we can remove/add create a token if we need it.

Reply:
Oh, I see, theres a "gap" at the top AND bottom... and they're different... 🤨
Still, odd that one has a descriptive variable (`gapChildPanel `) and the other is a literal.

Reply:
🤷‍♂